content/post/2017-05-22-using-asdf-vm.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Using ASDF-vm
+subtitle: Solve All of Your Version Problems
+date: 2017-05-21
+author: Connor Rigby
+draft: false
+tags: ["nerves"]
+---
+
+Recently there has been a number of breaking changes in the Erlang/Elixir world that have caused Nerves-based projects to fail to compile, causing a lot of confusion in slack. Follow along as we solve all your version inconsistencies with ASDF-vm.
```

Comment:
I think that we should file bugs against Elixir 1.4.4 if it is busted on OTP 19 so that the Elixir core team can fix them.  Do you remember some of the failures?